### DISCUSSION TITLE
no stream from YI Dome U2K

### DISCUSSION THREAD

**Discussion by Rene181970** [roleoroleo/yi-hack-Allwinner-v2#801] [Q&A] [answered]
Hi folks,

whereas the hack 0.3.0 seems to be installed right, I can not see a stream on chrome or with VLC media player.

> Links
>
> High res stream		rtsp://192.168.10.82/ch0_0.h264
> Only audio stream	rtsp://192.168.10.82/ch0_2.h264
> High res snapshot	http://192.168.10.82/cgi-bin/snapshot.sh?res=high&watermark=yes
> Low res snapshot	http://192.168.10.82/cgi-bin/snapshot.sh?res=low&watermark=yes
> GitHub project		yi-hack-Allwinner-v2
> 
> System
> 
> Hostname		YI
> Firmware Version	0.3.0
> Base Version		9.0.21.13_202109101104
> Model Suffix		b091qp
> Serial Number		SFUSY45VVSXZT0230518
> Local Time		Sun Jan 14 21:22:03 CET 2024
> Uptime			0h 26m 55s
> Load Average		2.49 3.52 2.87
> Free/Total Memory	8500/60904 KB
> Free SD Space	100%
> 
> Network
> 
> Address	192.168.10.82
> Netmask	255.255.255.0
> Gateway	192.168.10.1

![Guthub](https://github.com/roleoroleo/yi-hack-Allwinner-v2/assets/156612428/ee623b8c-e3f3-4696-8aea-420b3f07a0d1)

Messages from VLC:

> live555 error: Failed to connect with rtsp://192.168.10.82:554/ch0_0.h264
> main error: connection failed: Connection refused by peer
> satip error: Failed to connect to RTSP server 192.168.10.82:554
> main error: connection failed: Connection refused by peer
> access_realrtsp error: cannot connect to 192.168.10.82:554

Moving the cam works fine. May anybody help me please? It is my first IP Cam...

Rene

**Comment by roleoroleo** [comment, marked as answer]
I think your cam is a h51ga.
Not a b091qp.
Try with a fresh install.

**Comment by Rene181970** [comment]
Oh man! I ignored the version! Now it works. Thanks a lot!👍

**Reply by lebherz** [reply]
out of topic.. @Rene181970  interesting username 🤭
